src/utils/common.ts

```diff
@@ -601,3 +601,10 @@ export function deepMergeObjects(a: Object, b: Object) {
     }
   }
 }
+
+export function makeArray<T>(input: T | T[]): T[] {
+  if (!Array.isArray(input)) {
+    return [input];
+  }
+  return input;
```

Comment:
This way gives you a bit more power with typescript inference, using type predicates. It's a lot nicer, as it forces the return type of the method to be exactly the type that was put in, rather than a generic array. This preserves tuples, if the input was known to be a tuple. It's great.
```suggestion
export function makeArray<T extends any[]>(input: T): T;
export function makeArray<T>(input: T): T | [T] {
  return Array.isArray(input) ? input : [input];
```

Reply:
![image](https://github.com/user-attachments/assets/d2b20fa7-a010-4cfc-ae17-16b4442b0cd6)

Reply:
Tried a couple things but it didn't work, not sure how to fix it.

Reply:
I forgot to have the second overload when pasting here. Edited it now